File: src/Services/WareHouse/WareHouse.API/Application/Model/GetDataWareHouseBookBaseDTO.cs
using System.Collections.Generic;

namespace WareHouse.API.Application.Model
{
    public class GetDataWareHouseBookBaseDTO
    {
        public virtual IEnumerable<WareHouseItemDTO> WareHouseItemDTO { get; set; }
        public virtual IEnumerable<UnitDTO> UnitDTO { get; set; }
        public virtual IEnumerable<BaseSelectDTO> GetDepartmentDTO { get; set; }
        public virtual IEnumerable<BaseSelectDTO> GetEmployeeDTO { get; set; }
        public virtual IEnumerable<BaseSelectDTO> GetStationDTO { get; set; }
        public virtual IEnumerable<BaseSelectDTO> GetProjectDTO { get; set; }
        public virtual IEnumerable<BaseSelectDTO> GetCustomerDTO { get; set; }
        public virtual IEnumerable<BaseSelectDTO> GetAccountDTO { get; set; }
    }
}

File: src/WebClient/WareHouse/src/app/model/GetDataWareHouseBookBaseDTO.ts
import { BaseSelectDTO } from "./BaseSelectDTO";
import { UnitDTO } from "./UnitDTO";
import { WareHouseItemDTO } from "./WareHouseItemDTO";


export interface GetDataWareHouseBookBaseDTO {
    wareHouseItemDTO: WareHouseItemDTO[];
    unitDTO: UnitDTO[];
    getDepartmentDTO: BaseSelectDTO[];
    getEmployeeDTO: BaseSelectDTO[];
    getStationDTO: BaseSelectDTO[];
    getProjectDTO: BaseSelectDTO[];
    getCustomerDTO: BaseSelectDTO[];
    getAccountDTO: BaseSelectDTO[];
}